### ISSUE TITLE
it's a shame we can't enlarge the plugin window

### ISSUE THREAD

**Issue by Houston4444** [Stazed/non-mixer-xt#28]
The generic UI for plugins is limited in size, it's a shame, it would be nice to have the possibility to see as much parameters as the screen can. 
Here is an example with Plujain Ramp Live, the tremolo I have done years ago.

![Screenshot_20240314_084431](https://github.com/Stazed/non-mixer-xt/assets/3808689/e84095ad-604f-4b96-89b0-2bff780a2ae2)

**Comment by Stazed** [comment]
Yep, I want this also. But it will take a re-write of the generic UI since original non-mixer was never designed with larger plugins in mind, or resizing. The control pack used would display a graph sometimes and was also made for different widget types, but never worked well if you had more than 10 or 12 parameters.

I will get to this at some point, but running out of time for this project ATM.

**Comment by Stazed** [comment]
Well, just pushed a fix to the wip branch that looks like it will work for vertical resizing. No major re-write necessary.
It helps to look in the right place...

**Comment by Houston4444** [comment]
Cool, I tried, it works, thanks !
